vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -99,11 +99,11 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			errno = err.(syscall.Errno)
 			msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 			if retryCount >= maxRetryCount {
```

Comment:
True. With this fix, we are still exposed port being busy, will move the retry logic inside the attempt to bind and remove the higher level retry which needs #501 to be fixed.